libs/utils/env.py

```diff
@@ -410,50 +410,56 @@ def _init_target(self, force = False):
         if 'board' not in self.conf:
             self.conf['board'] = 'UNKNOWN'
 
+        board_name = self.conf['board'].upper()
```

Comment:
I like this simplification... can we just move them into a dedicated separate patch?
Just to keep it more clean the history and better isolate the patch which change the JUNO energy model... you can keep the patch either before or after... whatever is more convenient.

Reply:
Done.